cla_frontend/settings/base.py

```diff
@@ -215,18 +215,13 @@ def get_env_value(var_name):
         "simple": {"format": "%(levelname)s %(message)s"},
         "logstash": {"()": "logstash_formatter.LogstashFormatter"},
     },
-    "filters": {
-        "require_debug_false": {"()": "django.utils.log.RequireDebugFalse"},
-        "require_debug_true": {"()": "django.utils.log.RequireDebugTrue"},
-    },
     "handlers": {
-        "mail_admins": {
-            "level": "ERROR",
-            "filters": ["require_debug_false"],
-            "class": "django.utils.log.AdminEmailHandler",
-        }
+        "console": {"level": "INFO", "class": "logging.StreamHandler", "formatter": "simple", "stream": sys.stdout}
+    },
+    "loggers": {
+        "": {"handlers": ["console"], "level": "ERROR", "propagate": False},
+        "django.request": {"handlers": ["console"], "level": "ERROR", "propagate": False},
     },
-    "loggers": {"django.request": {"handlers": ["mail_admins"], "level": "ERROR", "propagate": True}},
```

Comment:
I can't see the replacement of the django request logger in `docker.py`

Reply:
Good catch, I've added it now